utils/web.py

```diff
@@ -216,74 +216,77 @@ def get_item_modifiers() -> tuple:
 
 def find_latest_update():
     """Search both local and remote versions, if different, prompt for update."""
-    # Get the list of releases from github, choose newest (even pre-release)
```

Comment:
All I did here was add try:     except Exception: logging.error("[!] Could not check for new update!") and moved the code inside the try, GitHub being weird..